scylladb/sphinx-scylladb-theme
# Add project link sidebar

Closes https://github.com/scylladb/sphinx-scylladb-theme/issues/936

## How to test

1. Build the docs.
2.  Open http://localhost:5500/
3. The project name in the sidebar should be clickable.
